### ISSUE TITLE
Fix EL major version naming scheme for Chef 12 packages.

### ISSUE THREAD

**Issue by johnbellone** [chef/omnibus-chef#336]
It seems with the release of Chef 12 the naming scheme of the packages (and metadata) for RHEL systems no longer includes the appropriate major version separation. The example I will use is *chef-12.0.3-1.x86_64.rpm* which is the name for _all of the Enterprise Linux packages_. The standard that the community has adopted is: chef-12.0.3-1.el5.x86_64.rpm, chef-12.0.3-1.el6.x86_64.rpm, etc. 

I am not quite sure the reasoning behind dropping the naming scheme. I *would* understand if the packages were actually the same, but EL5 is different than the EL6/EL7 packages. Please address!

**Comment by schisamo** [comment]
Related ticket with some background on the decision: https://github.com/chef/omnibus/issues/389

TL;DR - For most of our products we do build on EL5 and test/release on EL6 (and eventually EL7). Although the Chef packages don't do this right now they will begin doing it in the very near future.

**Comment by adamedx** [comment]
@manderson26, @stephen, is there a reason we've adopted the scheme we're currently using in Chef 12? Trying to understand if this is just a cosmetic change for us or if it has deeper implications.

**Comment by lamont-granquist** [comment]
I think we adopted the official RHEL distro rpm format, but the community has the .elX.arch.rpm format which is a defacto standard, and we're fighting against that.

**Comment by pburkholder** [comment]
Looks like this can be closed since this, https://github.com/chef/omnibus/pull/424, was merged on Feb 7 and should have restored the Fedora-style distro tag to the RPMs. E.g., you can now download https://opscode-omnibus-packages.s3.amazonaws.com/el/6/x86_64/chef-12.2.1-1.el6.x86_64.rpm.
